hack/update-helm.sh

```diff
@@ -74,6 +74,10 @@ EOF
 search_service_line="spec:"
 replace_service_line=$(
   cat <<'EOF'
+{{- if .Values.webhookService.ipDualStack.enabled }}
+  ipFamilies: {{ toYaml .Values.webhookService.ipDualStack.ipFamilies | nindent 4 }}
+  ipFamilyPolicy: {{ .Values.webhookService.ipDualStack.ipFamilyPolicy }}
+{{- end }}
   type: {{ .Values.webhookService.type }}
```

Comment:
nit
```suggestion
  {{- if .Values.webhookService.ipDualStack.enabled }}
  ipFamilies: {{ toYaml .Values.webhookService.ipDualStack.ipFamilies | nindent 4 }}
  ipFamilyPolicy: {{ .Values.webhookService.ipDualStack.ipFamilyPolicy }}
  {{- end }}
  type: {{ .Values.webhookService.type }}
```
()the indent change will be propagated  to charts)

Reply:
Thanks. Fixed